src/shared/components/msktable/MSKTable.tsx

```diff
@@ -20,13 +21,14 @@ export type Column<T> = {
     render:(data:T)=>JSX.Element;
     download?:(data:T)=>string;
     tooltip?:JSX.Element;
+    firstSortDirection?:SortDirection;
```

Comment:
What about defaultSortDirection?  I agree firstSortDirection is kind of strange.